src/index.ts

```diff
@@ -1,13 +1,7 @@
 import { Chess } from '../cjsmin/src/chess';
 import { gameChunks } from './fileReader';
-import { KDRatioMetric, MateAndAssistMetric } from './metrics/captures';
-import { MoveDistanceMetric } from './metrics/distances';
-import { MetadataMetric } from './metrics/misc';
-import {
-  GameWithMostMovesMetric,
-  PieceLevelMoveInfoMetric,
-} from './metrics/moves';
-import { PromotionMetric } from './metrics/promotions';
```

Comment:
can we just comment these out

Reply:
i guess, but adding imports back not that hard